src/exchanges/fetch.ts

```diff
@@ -75,9 +75,19 @@ const createFetchSource = (operation: Operation) => {
         abortController !== undefined ? abortController.signal : undefined,
     };
 
+    const startTime = Date.now();
```

Comment:
Any reason as to why you're using Date.now over performance.now?

Reply:
No particular reason. Is there an advantage to using `performance.now` other than sub-millisecond precision?

Reply:
It’s relative to page load in comparison to unix epoch. I think this is a minimal difference, I did read that it’s not available in opera mini and IE9 so I don’t know if that’s a blocker. It’s up to you, what you prefer was just asking if there was a reason. I think either is good for the job but perf.now will probably be a micro optimisation since it revolves around a smaller number.